metatests/integration_test.sh

```diff
@@ -18,8 +42,13 @@ mkdir results
 # Print the autograder results for easy inspection
 cat results/results.json
 
+ls submission/
```

Comment:
Is this only for debugging?  It might make sense to do the ls with some message, but then again it only helps for a very specific type of integration test, so it's probably best off not to print it at all.

Reply:
Yeah, given the latter point I'll remove it.